app/services/force/soql/preference_service.rb

```diff
@@ -19,20 +19,25 @@ def app_preferences_for_application(application_id)
 
       def app_preferences_for_listing(opts)
         query_scope = app_preferences_for_listing_query(opts)
-        puts query_scope
+
         query_scope.query
       end
 
       private
 
       def buildAppPreferencesFilters(opts)
+        if opts[:accessibility]
+          accessibility_filters = opts[:accessibility].split(', ')
```

Comment:
I'd like a test for this piece to make it clearer what the expected input and output are if possible. Maybe break it into a separate function and add a unit test? 
A comment might be helpful here too.